common/weave/client.go

```diff
@@ -251,3 +251,7 @@ func weaveCommand(arg ...string) exec.Cmd {
 	cmd.SetEnv(append(os.Environ(), "DOCKER_API_VERSION="+dockerAPIVersion))
 	return cmd
 }
+
+func errorf(format string, a...interface{}) error {
+	return fmt.Errorf(format + ". If you are not running Weave Net, you may wish to suppress this warning by launching scope with the `--weave=false` option.", a...)
```

Comment:
This package is the wrong place for that message. We have no idea that we are being called in the context of the scope app or probe here.

A possible fix here is to define a special error type, so that call sites can then distinguish between errors that may be due to Weave Net not being there, and others.